node/runtime/common/src/lib.rs

```diff
@@ -133,3 +133,6 @@ frame_support::parameter_types! {
 		.max
 		.get(DispatchClass::Normal);
 }
+
+// Accounding to the evm gas benchmark, 1 gas ~= 40_000 weight.
+pub const WEIGHT_PER_GAS: u64 = 40_000;
```

Comment:
Need review and double check.

Reply:
Yeah, this is a critical variable. The value I choose considering the current EVM benchmark file and other project configurations. We can upgrade to the pangolin test network firstly and do more tests about this. If necessary, update later.